ipie/addons/eph/__init__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 The ipie Developers. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Directory for additions to ipie which depend on the core ipie library.
+# New features should mirror the ipie layout e.g.
```

Comment:
Remove this additional part from the init. The "Directory for additions to ipie ...." comment was only meant to be in the top level `__init__`